src/TriggerService/AzureStorageUtility.cs

```diff
@@ -81,8 +82,8 @@ private static async Task<string> GetStorageAccountKeyAsync(StorageAccountInfo s
             return (await storageAccount.GetKeysAsync())[0].Value;
         }
 
-        private static Task<string> GetAzureAccessTokenAsync(string resource = "https://management.azure.com/")
-            => new AzureServiceTokenProvider().GetAccessTokenAsync(resource);
+        public static async Task<string> GetAzureAccessTokenAsync(string scope = "https://management.azure.com/.default")
```

Comment:
Does this work?  There is an intentional double slash usually: https://github.com/Azure/azure-sdk-for-net/blob/bc9f38eca0d8abbf0697dd3e3e75220553eeeafa/sdk/identity/Azure.Identity/src/AzureAuthorityHosts.cs#L53

Reply:
Yeah, single slash works. Good to know the convention is double slash for public cloud though.